fastlane/lib/fastlane/documentation/markdown_docs_generator.rb

```diff
@@ -134,15 +134,16 @@ def generate!(target_path: nil)
       require 'yaml'
       FileUtils.mkdir_p(target_path)
       docs_dir = File.join(target_path, "docs")
+      generated_actions_dir = File.join("generated", "actions")
```

Comment:
Does this directory need a `FileUtils.mkdir_p` on it? 🤔

Reply:
I think that you're right, it wouldn't be needed on related docs branch.